### PR TITLE
Planck keymap and README update

### PR DESCRIPTION
**Pull request by yttyx**
## Description

Updated planck keymap to match yttyx niu mini keymap. Tidied README file.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/planck/keymaps/yttyx/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## To build
+
+```
+make planck/rev6:yttyx
+```
+
+## To build and flash
+
+```
+make planck/rev6:yttyx:dfu-util
```
Comment (drashna) [inline keyboards/planck/keymaps/yttyx/README.md:15]: You could even use:
```suggestion
make planck/rev6:yttyx:flash
```

Comment (yttyx) [reply keyboards/planck/keymaps/yttyx/README.md]: Fixed

**Comment by yttyx** [comment]
Hi Drashna - thanks again for the review. I have also made a change suggested by fauxpark in my Planck PR (https://github.com/qmk/qmk_firmware/pull/9190).

**Comment by zvecr** [comment]
Thanks!

**Comment by yttyx** [comment]
Thank you!

**Pull request merged**